src/modules/holder/pages/Home/Home.tsx

```diff
@@ -24,37 +30,76 @@ export const Home: FC = () => {
     return (
       <>
         <Header title="Your tickets" />
-        <Container fullWidth>
+        <Container fullWidthCenter>
           {error && <Typography variant="e1">{error?.message}</Typography>}
         </Container>
       </>
     )
   }
 
-  if (data.length === 0) {
+  const tickets = data.filter((credentialItem) => {
+    const credentialSchema = (credentialItem as StoredW3CCredential).credentialSchema
+    return credentialSchema?.id === JSON_SCHEMA_URL
+  })
+
+  if (tickets.length === 0) {
     return (
       <>
         <Header title="Your tickets" />
-        <Container fullWidth>
+        <Container fullWidthCenter>
           <Typography variant="p2">You don't have any tickets yet.</Typography>
+          <S.IconContainer>
+            <NoTicket />
+          </S.IconContainer>
         </Container>
       </>
     )
   }
+
+  // @ts-ignore
+  const validTickets: StoredW3CCredential[] = tickets.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    return Date.parse(credentialSubject?.startDate) >= Date.now()
+  })
+
+  // @ts-ignore
+  const expiredTickets: StoredW3CCredential[] = tickets.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    return Date.parse(credentialSubject?.startDate) < Date.now()
+  })
+
+  const getCredential = (credentialSubject: AnyData, credentialItem: StoredW3CCredential) => {
+    const credential: Credential = {
+      title: credentialSubject?.eventName,
+      date: format(new Date(credentialSubject?.startDate), 'dd.MM.yyyy'),
+      time: format(new Date(credentialSubject?.startDate), 'HH:mm'),
+      credentialId: credentialItem?.id,
+    }
+
+    return credential
+  }
+
+  const getTicketCards = (params: { tickets: StoredW3CCredential[]; isValid: boolean }) => {
+    const { tickets, isValid } = params
+    return tickets.map((credentialItem: StoredW3CCredential) => {
+      const credentialSubject = credentialItem?.credentialSubject
+      const credential = getCredential(credentialSubject, credentialItem)
+      return <TicketCard key={credentialItem.id} credential={credential} isValid={isValid} />
+    })
+  }
```

Comment:
u can drop `getCredential`

```suggestion
  const getTicketCards = ({ tickets, isValid } : { tickets: StoredW3CCredential[]; isValid: boolean }) =>
    tickets.map((credentialItem: StoredW3CCredential) => {
      const credential = {
        title: credentialSubject?.eventName,
        date: format(new Date(credentialSubject?.startDate), 'dd.MM.yyyy'),
        time: format(new Date(credentialSubject?.startDate), 'HH:mm'),
        credentialId: credentialItem?.id,
      }

      return <TicketCard key={credentialItem.id} credential={credential} isValid={isValid} />
    })
```